README.md

```diff
@@ -654,6 +657,32 @@ An example response for all `POST` commands:
 }
 ```
 
+### Send POI
+
+Sending a POI to the car.
+
+* `POST /webapi/v1/user/vehicles/:VIN/snedpoi`
+    * Where `:VIN` is your vehicle's VIN.
+    * Remember to include the `Authorization: Bearer` header.
+    * Remember to include the `Content-Type: application/x-www-form-urlencode` header.
+
+#### Body
+```
+body={
```

Comment:
Do you know which of these fields are optional?  Can I put in a City without a lat/long? What happens if I put in a postal code, but use the wrong street name?